.circleci/config.yml

```diff
@@ -172,7 +171,7 @@ jobs:
           ELASTICSEARCH_PORT: 9200
           DJANGO_AWS_S3_REGION_NAME: eu-west-1
       # services
-      - image: circleci/postgres:12-ram
+      - image: circleci/postgres:17-ram
```

Comment:
This tag does not seem to exist on DockerHub
And if I remember correctly, the `circleci/` images are deprecated, should we take this opportunity to upgrade to `cimg/` images?

Reply:
There's no RAM variant anymore, but should pass with `cimg/postgres:17.0`